scripts/deploy-mainnet.js

```diff
@@ -1,11 +1,14 @@
 const { ethers } = require("hardhat");
 const fs = require("fs");
 const { FOUNDATION_MULTISIG } = require("../constants/shared.js");
+const  abi  = require("../constants/abi.js");
```

Comment:
Can we instead attach the contract address to an existing factory?
This is the method for it https://docs.ethers.io/v5/api/contract/contract-factory/#ContractFactory-attach

Reply:
Yes it's possible, so i delete the abi.js file and i create the attach function
https://github.com/wagmiog/exchange-contracts/blob/feature/monorepo/scripts/deploy-mainnet.js#L96